monitor/census.go

```diff
@@ -1187,6 +1213,24 @@ func SegmentDownloaded(ctx context.Context, nonce, seqNo uint64, downloadDur tim
 	}
 }
 
+func SegSceneClassificationResult(ctx context.Context, seqNo uint64, class string, prob float64) {
+	clog.V(logLevel).Infof(ctx, "Logging SegSceneClassificationResult... class=%s prob=%v", class, prob)
+	if err := stats.RecordWithTags(census.ctx,
+		manifestIDTag(ctx, tag.Insert(census.kSegClassName, class)),
+		census.mSegmentClassProb.M(prob)); err != nil {
```

Comment:
Nice, I like the idea of having one metric for classification probability and segmenting classification type by tag